Reenable litex tests

rw1nkler opened the pull request:
This PR reenables litex tests. To fix the litex example we need to:
- update symbiflow toolchain packages
- split the examples into multiple jobs in Travis CI to match with the Travis build time limit

The changes from https://github.com/SymbiFlow/symbiflow-examples/pull/71 are included in this PR
Closes https://github.com/SymbiFlow/symbiflow-examples/pull/71

scted commented:
tried two of the tests using this pull request on ubuntu:focal ... both failed
 litex on line 84 of symbiflow.py
![image](https://user-images.githubusercontent.com/2945762/100362091-d1c40980-2faf-11eb-9622-e04b48de30a0.png)
 
counter_test failing pushd xc7/counter_test after synthesis
![image](https://user-images.githubusercontent.com/2945762/100362388-34b5a080-2fb0-11eb-8bd4-02e36cccae73.png)

pgielda commented:
You seem to be using old conda env and have old symbiflow tools. You would have to do it completely from scratch.

In any case lets wait for Travis run.

We are separately going to introduce Github Actions soon so I would suggest waiting for that...

rw1nkler commented:
@scted Have you updated the conda environment with the latest packages introduced in this PR? If not you have to remove the existing environment and recreate it once again.

scted commented:
i was running the .travis.ci ... everything (conda, symbiflow) was supposedly installed and run in a virgin ubuntu container ... not sure what i'm doing wrong ... will look for a cause on my end

rw1nkler commented:
@scted We updated our CI. Now it is possible to grab the output bitstream directly from the GitHub Actions:
https://github.com/SymbiFlow/symbiflow-examples/actions/runs/385736139

I downloaded the `litex_demo` and tested it on hardware. It works correctly.
The CI executes the instructions from the README file. I also added additional information to the CI logs. Maybe it will be helpful to you.

scted commented:
thanks ... got the flow working on my own machine as well using the current /master branch

rw1nkler commented:
Great!
